app/views/users/_app_permissions.html.erb

```diff
@@ -0,0 +1,36 @@
+<table class="table table-bordered table-striped table-on-white">
+  <thead>
+    <tr class="table-header">
+      <th>Application</th>
+      <% unless user_object.api_user? %>
+          <th>Has access?</th>
+      <% end %>
+      <th><%= "Other" unless user_object.api_user? %> Permissions</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @application_permissions.each do |application, permissions| %>
+      <% if permissions.present? %>
+        <tr>
+          <td>
+            <% if application.retired? %>
+                <del><%= application.name %></del>
+            <% else %>
+                <%= application.name %>
+            <% end %>
+          </td>
+          <% if user_object.api_user? %>
+            <td></td>
+          <% else %>
+            <td>
+              <%= user_object.has_access_to?(application) ? "Yes" : "No" %>
+            </td>
+          <% end %>
+          <td>
+            <%= permissions.join(', ') %>
```

Comment:
We probably want to remove `signin` from here as we're handling it in the previous column.  If we leave it in then we have a disconnect between the editable UI where `signin` won't appear under "Other permissions" and this list where it will.